javascript/code-challenges/__tests__/binary.test.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+const binary = require('../array-binary-search.js');
```

Comment:
should be `const binary = require('../arrayBinarySearch/array-binary-search.js'); ` because the file is inside a folder